teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission = nil)
```

Comment:
- [ ] Please check once more the optional value for the parent_permission parameter. It takes a boolean value(`true`) as the default value :+1: